packages/dds/matrix/src/matrix.ts

```diff
@@ -630,12 +711,31 @@ export class SharedMatrix<T = any>
 
 				if (local) {
 					// We are receiving the ACK for a local pending set operation.
-					const { rowHandle, colHandle, localSeq } = localOpMetadata as ISetOpMetadata;
-
-					// If this is the most recent write to the cell by the local client, remove our
-					// entry from 'pendingCliSeqs' to resume allowing remote writes.
-					if (this.isLatestPendingWrite(rowHandle, colHandle, localSeq)) {
-						this.pending.setCell(rowHandle, colHandle, undefined);
+					const { rowHandle, colHandle } = localOpMetadata as ISetOpMetadata;
```

Comment:
While "resolveConflict" should be raised only for local client, I'd expect all the other logic to be exactly the same whether it's local changes or not. Why do we have so much unique logic on those two paths? We might have more data (rowHandle, colHandle), but wouldn't we want to validate that remote path (with this.rows.adjustPosition() logic) arrives to exactly same answer (handle)?

Same question / suggesting as in other place - I think tracking only ref seq number is not enough. We need to track clientId of a client who changes cell last time, and if it's same client, then changes should be allowed.

Reply:
Deduplicated the code.